va/va_backend.h

```diff
@@ -197,8 +197,14 @@ struct VADriverVTable
 	VAStatus (*vaQuerySurfaceError) (
 		VADriverContextP ctx,
 		VASurfaceID render_target,
-                VAStatus error_status,
-                void **error_info /*out*/
+        VAStatus error_status,
+        void **error_info /*out*/
+	);
+
+	VAStatus (*vaQueryCenc) (
+        VADriverContextP ctx,
+        VABufferID buffer,
+        VACencStatusBuf *info/*in, out*/
```

Comment:
Also, I guess, here the indentation is wrong.

Reply:
Accept